server/src/routes/v1/apps/handlers/createAppVersion.js

```diff
@@ -118,87 +120,80 @@ module.exports = {
 
         let versionId = null
```

Comment:
I think this can be removed and its sole use can be replaced with `appVersion.id`